tensorboard/plugins/histogram/metadata.py

```diff
@@ -36,13 +36,14 @@ def create_summary_metadata(display_name, description):
   Returns:
     A `tf.SummaryMetadata` protobuf object.
   """
-  content = HistogramMetadata()
-  metadata = tf.SummaryMetadata(display_name=display_name,
-                                summary_description=description)
-  metadata.plugin_data.plugin_name = PLUGIN_NAME
-  content = json.dumps(content._asdict())  # pylint: disable=protected-access
-  metadata.plugin_data.content = content
-  return metadata
+  content = json.dumps(
+      HistogramMetadata()._asdict())  # pylint: disable=protected-access
```

Comment:
If you want this to be on one line, it's fine: `pylint: disable`s are allowed to exceed 80 characters. Also fine as is.